django/db/backends/mysql/features.py

```diff
@@ -305,6 +305,9 @@ def supports_transactions(self):
         """
         return self._mysql_storage_engine != "MyISAM"
 
+    uses_savepoints = property(operator.attrgetter("supports_transactions"))
```

Comment:
As both features use support_transactions, can't we make a support_transactions feature?

Reply:
Sorry. Can't we directly use `supports_transactions` feature flag?